rezz.lic

```diff
@@ -45,7 +45,7 @@ class Rezz
                echo "Using default values for REJUV, make a 'rezz'=>Rejuvenation waggle set to avoid this."
                get_data('spells').spell_data['Rejuvenation']
              end
-
+    @total_infused = 0
```

Comment:
This could be put above under settings, etc. It's a little out of place, most variables should be initialized at the top of a method for clarity. I know @rezz and @rejuv are also variables being initialized, but this is a single one that can go with the others.